App/UI/Home/Cells/HomeButtonCell.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  HomeButtonCell.swift
+//  DebugMenu
+//
+//  Created by Paride on 05/06/2020.
+//
+
+import Extensions
+import Foundation
+import Lottie
+import Tempura
+
+struct HomeButtonCellVM: ViewModel {
```

Comment:
I'd change the name of this cell as it looks very broad and reusable, while it is very specific instead